package.json

```diff
@@ -74,6 +74,27 @@
             "type": "object",
             "title": "Rust configuration",
             "properties": {
+                "rls.client.showStderrInOutputChannel": {
```

Comment:
How about `showServerStdErr` to emphasize that it's server's stderr that will be shown?